playground/backend/configs/DEFAULT_EXAMPLES.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I am not sure that this file should be in `configs` folder. @KhaninArtur @ilya-kozyrev what do you think?

Reply:
I think it's a good place for this file. What place do you propose?

Reply:
Now `configs` folder contains files that use to configure each SDK (commands for `compile` and `run` steps). But this file contains constants for each SDK, so maybe it would be better to create constant values into the code and do not create one more `config` file?

Reply:
Hm, on the other hand, we can configure default examples from one place instead of looking for them in the code. What do you think about adding the default_example field to the corresponding `.json` config? E.g. add `"default_example": "SDK_GO/MinimalWordCount"`, to the `SDK_GO.json` file.

Reply:
Sounds good.

Reply:
@pavel-avilov could you please change the configs based on the discussion above?

Reply:
Done.